src/main/app/form/validation.ts

```diff
@@ -28,7 +28,14 @@ export const isFieldFilledIn: Validator = value => {
 };
 
 export const atLeastOneFieldIsChecked: Validator = fields => {
```

Comment:
@rahulsanjava19 Will this not impact others using the function?
If yes, can we can add it as new function for c100 please?

Reply:
we filtered values to remove empty values so it won't break. Existing test cases passed as well where this functions is used